test/integration/com/unifina/service/EthereumServiceIntegrationSpec.groovy

```diff
@@ -0,0 +1,43 @@
+package com.unifina.service
+
+import com.unifina.domain.security.IntegrationKey
+import com.unifina.domain.security.SecUser
+import grails.test.spock.IntegrationSpec
+
+class EthereumServiceIntegrationSpec extends IntegrationSpec {
+	void "hasEthereumAddress query works with lowercase or uppercase"() {
+		setup:
```

Comment:
Please add similar test for getEthereumUser, if not already present

Reply:
Done